server/bin/www

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env node
+
+/**
+ * Module dependencies.
+ */
+var app = require('../app');
+var debug = require('debug');
+var http = require('http');
+var amqp = require('amqplib/callback_api');
+var io = require('socket.io');
+
+/**
+ * Get port from environment and store in Express.
+ */
+var port = normalizePort(process.env.PORT || 4000);
+app.set('port', port);
+
+/**
+ * Create HTTP server.
+ */
+var server = http.createServer(app);
+
+/**
+ * Listen on provided port, on all network interfaces.
+ */
+server.listen(port);
+server.on('error', onError);
+server.on('listening', onListening);
+
+/* Setup socket.io stuff for live webpage */
+io = io.listen(server);
+
+/* Setup the AMQP channel with RabbitMQ (change host if needed) */
+var rmqport = '5672';
+amqp.connect('amqp://localhost:' + rmqport, function(err, conn) {
+  try {
+    conn.createChannel(function(err, ch) {
+      // NOTE: the string in this variable should be 
+      // the same in the provider
+      var q = 'msgQueue';
+      var ex = 'hermesExchange';
+  
+      // check if queue is valid and wait for messages
+      ch.assertExchange(ex, 'fanout', {durable: false});
+      ch.assertQueue(q, {durable: false});
+      ch.bindQueue(q.queue, ex, '');
+      console.log("Express: waiting for messages in %s", q);
+      ch.consume(q, function(msg) {
+        var msgString = JSON.stringify(msg);
```

Comment:
For me, this returned a bunch of numbers on my console. Try doing `msg.content.toString()`!